labelbox/schema/foundry.py

```diff
@@ -0,0 +1,160 @@
+from datetime import datetime
+from typing import Any, Dict, Optional, Union
+from pydantic import BaseModel
+from labelbox import exceptions
+from labelbox.schema.identifiables import DataRowIds, GlobalKeys
+from labelbox.schema.task import Task
+from labelbox.utils import _CamelCaseMixin
+
+
+class App(_CamelCaseMixin, BaseModel):
+    id: Optional[str]
+    model_id: str
+    name: str
+    description: str
+    inference_params: Dict[str, Any]
+    class_to_schema_id: Dict[str, str]
+    created_by: Optional[str] = None
+
+    @classmethod
+    def type_name(cls):
+        return "App"
+
+
+class Model(_CamelCaseMixin, BaseModel):
+    id: str
+    description: str
+    inference_params_json_schema: Dict
+    name: str
+    ontology_id: str
+    created_at: datetime
+
+
+APP_FIELD_NAMES = list(App.schema()['properties'].keys())
+MODEL_FIELD_NAMES = list(Model.schema()['properties'].keys())
+
+
+class FoundryClient:
+
+    def __init__(self, client):
+        self.client = client
+
+    def _create_app(self, app: App) -> App:
+        query_str = """
+            mutation CreateDataRowAttachmentPyApi(
+                $name: String!, $modelId: ID!, $description: String, $inferenceParams: Json!, $classToSchemaId: Json!
+            ){
+                createModelFoundryApp(input: {
+                    name: $name 
+                    modelId: $modelId 
+                    description: $description
+                    inferenceParams: $inferenceParams
+                    classToSchemaId: $classToSchemaId
+                })
+                {
+                    id
+                    name
+                    modelId
+                    description
+                    inferenceParams
+                    classToSchemaId
+                }
+            }
+            """
+
+        params = app.dict(by_alias=True, exclude={"id"})
```

Comment:
any reason not to use `APP_FIELD_NAMES` like we do below?

Reply:
good idea, I'll update.